src/OOVPADatabase/OOVPA.h

```diff
@@ -198,120 +229,236 @@ typedef enum _eDBScanType {
+#define REGISTER_OOVPA_16(Symbol, Version, ...) REGISTER_OOVPA(Symbol, Version), MSVC_EXPAND(REGISTER_OOVPA_15(Symbol, __VA_ARGS__))
+#define REGISTER_OOVPA_17(Symbol, Version, ...) REGISTER_OOVPA(Symbol, Version), MSVC_EXPAND(REGISTER_OOVPA_16(Symbol, __VA_ARGS__))
+#define REGISTER_OOVPA_18(Symbol, Version, ...) REGISTER_OOVPA(Symbol, Version), MSVC_EXPAND(REGISTER_OOVPA_17(Symbol, __VA_ARGS__))
+#define REGISTER_OOVPA_19(Symbol, Version, ...) REGISTER_OOVPA(Symbol, Version), MSVC_EXPAND(REGISTER_OOVPA_18(Symbol, __VA_ARGS__))
 
-// clang-format off
 #define OV_BYTES_0(...)
- #define OV_BYTES_1(Offset, Value) OV_BYTE(Offset, Value)
- #define OV_BYTES_2(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_1(Offset+1, __VA_ARGS__))
- #define OV_BYTES_3(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_2(Offset+1, __VA_ARGS__))
- #define OV_BYTES_4(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_3(Offset+1, __VA_ARGS__))
- #define OV_BYTES_5(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_4(Offset+1, __VA_ARGS__))
- #define OV_BYTES_6(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_5(Offset+1, __VA_ARGS__))
- #define OV_BYTES_7(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_6(Offset+1, __VA_ARGS__))
- #define OV_BYTES_8(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_7(Offset+1, __VA_ARGS__))
-// clang-format on
+#define OV_BYTES_1(Offset, Value)      OV_BYTE(Offset, Value)
+#define OV_BYTES_2(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_1(Offset + 1, __VA_ARGS__))
+#define OV_BYTES_3(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_2(Offset + 1, __VA_ARGS__))
+#define OV_BYTES_4(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_3(Offset + 1, __VA_ARGS__))
+#define OV_BYTES_5(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_4(Offset + 1, __VA_ARGS__))
+#define OV_BYTES_6(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_5(Offset + 1, __VA_ARGS__))
+#define OV_BYTES_7(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_6(Offset + 1, __VA_ARGS__))
+#define OV_BYTES_8(Offset, Value, ...) OV_BYTE(Offset, Value), MSVC_EXPAND(OV_BYTES_7(Offset + 1, __VA_ARGS__))
+
+#define CALL_unk   call_unknown
+#define CALL_cde   call_cdecl
+#define CALL_std   call_stdcall
+#define CALL_thi   call_thiscall
+#define CALL_fas   call_fastcall
+#define CALL(Name) CALL_##Name
+
+// For generate symbol's suffix name, mainly for registers, and extended API usage.
+#define PARAM(Param, Name)            Param, Name
+#define PARAM1(Param)                 Param, ""
+#define PARAM_TOKEN_unk(Index, Name)  _unk##Index
+#define PARAM_TOKEN_void(Index, Name) // No argument, do not append to symbol reference.
+#define PARAM_TOKEN_stk2(Index, Name) // (Custom) argument is stored in call stack with two stacks, do not append to symbol reference.
```

Comment:
This comment now makes it sound like there are two stacks, which is not true since every thread has only one stack. It should just say that this is the second push done for an argument passed with the stack.